setup.py

```diff
@@ -534,14 +534,15 @@ def run(self):
 ]
 _package_data['deluge.ui.gtk3'] = ['glade/*.ui']
 
-setup_requires = ['setuptools', 'wheel']
+setup_requires = ['setuptools', 'wheel', 'py2app']
 install_requires = [
     'twisted[tls]>=17.1',
     # Add pyasn1 for setuptools workaround:
     #   https://github.com/pypa/setuptools/issues/1510
     'pyasn1',
     'rencode',
     'pyopenssl',
+    'pygobject',
```

Comment:
This is not a requirement for minimal running since this does not apply to `deluged`. 

This does raise the question that do potentially other ui-specific requirements need scrutiny e.g. mako

It could however be included under extra_requires if needed

Reply:
I'm not terribly familiar with python setuptools. What would be the preferred way to have this "just work®", adding it under extra_requires as you say? I worked on this quite a long time ago, but if I added this it's most likely because I got an error without it along the way.

Reply:
Yes, `mako` is included into `install_requires` but is WebUI-specific only.